vault/dendron.alternatives.foam-vs-dendron.md

```diff
@@ -0,0 +1,30 @@
+---
+id: p5fMTi-6zOyX1TwhL6dM0
+title: Foam Vs Dendron
+desc: ''
+updated: 1626057882250
+created: 1626056573800
+---
+
+> Dendron is your IDE for General Knowledge, see how it stacks up vs. Foam.
+
+Foam is a personal knowledge management system built as a collection of various tools that can expand its capabilities. Like Dendron, Foam is built on top of VSCode and can benefit from the endless list of extensions existing in the ecosystem. Like Dendron, Foam is open source and local-first. 
+
+## Dendron has more governance over core features.
+- Foam depends on a couple of extensions that provide a significant chunk of Foam's capabilities:
```

Comment:
not a big deal for this one but a potential improvement is to add these blocks as more generic "features" in dendron.features and then reference it from here. I don't think it's immediately possible since the paragraphs are written pretty specifically to Foam but maybe for later.